Diagnostics/Get-VSwitchPathCost.ps1

```diff
@@ -88,98 +66,82 @@ function Log($Message)
     }
 }
 
+$cpuInfo = @(Get-WmiObject -Class win32_processor -Property NumberOfCores, MaxClockSpeed, NumberOfLogicalProcessors)
```

Comment:
Can you check to see if there's a property for currentClockSpeed, and throw an error if MaxClockSpeed != CurrentClockSpeed?  In the BIOS, by default, the machine uses a PowerSaving profile, and the CPU speed is much lower than current clock speed.

We want the power profile to be "Performance", when doing perf measurements, so the CPU speed doesn't vary.